internal/repo/helpers/loginService.go

```diff
@@ -0,0 +1,21 @@
+package helpers
```

Comment:
Don't see the use for this package. Remove if not required.

Reply:
done